app/controllers/application_controller.rb

```diff
@@ -66,7 +66,7 @@ def status
   helper_method :sublayout
 
   def redirect_back_or_to(*fallback_params)
-    redirect_to :back
+    redirect_back(fallback_location: root_path)
   rescue ActionController::RedirectBackError
     redirect_to(*fallback_params)
```

Comment:
Can `redirect_back` with `fallback_location` as specified above, raise `ActionController::RedirectBackError`?

Reply:
May be not, because I am really confused about the fallback path from application controller, So checked with root_path which also make fails  2 test cases, redirect_to(*fallback_params) is also not a solution as it will also make fails the test case.

Any suggestion what can be the correct fallback_location?

Reply:
If it was me, I'd try to see what happens when `fallback_location` parameter is not set. Also I'd add a breakpoint to see what these `fallback_params` are and whether they can be passed as `fallback_location`.

Reply:
I tried all but nothing worked even setting fallback_location to fallback_params

Reply:
I think we don't need to do any change to this method, because `ActionController::RedirectBackError` will rescue the error and this will not fail so reverting back to how its was instead of giving any fallback location.

Reply:
@akostadinov No I don't think `redirect_back` is supposed to raise such error because that's the reason why it was created. `fallback_location` is a required argument now.

Reply:
@nidhi-soni1104 You should redirect_back to the route passed in `*fallback_params`. When no route is passed, then use `root_path` as default.

Reply:
This was my guess, that `fallback_params` should contain information where to fallback to. Thank you.